engine/input/proto/input/input_ddf.proto

```diff
@@ -183,7 +183,8 @@ enum Gamepad
     GAMEPAD_GUIDE = 24;
     GAMEPAD_CONNECTED = 25;
     GAMEPAD_DISCONNECTED = 26;
-    MAX_GAMEPAD_COUNT = 27;
+    GAMEPAD_RAW = 27;
```

Comment:
The trigger to set in the gamepad section of your input bindings to get raw gamepad data from a connected gamepad. With raw data I mean the axis, hats and buttons as they are read from the OS, without applying any deadzone or other operations on the data.